eox_tagging/api/v1/serializers.py

```diff
@@ -38,9 +38,9 @@ class Meta:  # pylint: disable=old-style-class, useless-suppression
     def get_meta(self, instance):
         """Getter of read-only field that returns technical information."""
         return {
-            "target_id": instance.target_object_id,
+            "target_id": str(instance.target_object),
```

Comment:
could you include the test for the meta response  in a **different** pr ?

Reply:
yes!